src/core/util/ElfReader.hpp

```diff
@@ -163,17 +231,24 @@ class ElfReader {
 	void setup(const char*);
 	bool process_symboltable(FILE *fp, Elf64_Ehdr const *ehdr, int sect_num);
 	int process_section(Elf64_Shdr *sect_hdr, char *sect_name_buff);
+	void process_segment(Elf64_Phdr *seg_hdr);
 
-	// Returns true if it finds a valid ELF header.  Stores ELFCLASS32 or 64 in m_elfclass.
```

Comment:
why

Reply:
Naja, das führt zu einer vereitlichen direkt nach dem auslesen des ELFs. Nachdem ein 32 Bit Segment immer in ein 64 bit segment passt, find ich das garnicht SO wahnsinnig daneben. Irritiert hat es mich aber auch.

Reply:
Nee, das war nicht mein Problem; das war IIRC wieder ein reiner Whitespace-Change.

Reply:
Aso ja. Ist aber nur innerhalb des Patchsets.